### PR TITLE
snmp.yml: Add Aruba wireless controllers

### PR DESCRIPTION
**Pull request by RichiH**
MIBs can be found at http://support.arubanetworks.com/default.aspx behind paywall/registration. There is no legal way to have world-readable access to the files.

### CODE REVIEW

**Comment by RichiH** [comment]
@juliusv @SuperQ: FYI.

This is without string support, and without matching up AP names via MAC address.

File: snmp.yml [hunk 1]
```diff
@@ -1644,3 +1644,1071 @@ servertech_sentry3:
           type: Integer32
         - labelname: outletIndex
           type: Integer32
+
+aruba_controller
```
Comment (SuperQ) [inline snmp.yml]: `aruba_controller:`

**Comment by RichiH** [comment]
Thanks; I pushed a fixed version.

**Comment by SuperQ** [comment]
It might be nice to simplify the `walk:` section.  But if it's too much of a performance hit to walk larger trees, this is fine.

**Comment by RichiH** [comment]
I don't have data on that and the test servers are down. I suggest we play with that during 33c3 and merge a cleanup after testing it.

**Comment by brian-brazil** [comment]
The generator is at a PoC stage at https://github.com/prometheus/snmp_exporter/tree/generator, all future additions will be via it.

**Comment by brian-brazil** [comment]
Please resubmit using the generator.

**Pull request closed without merging**